wix-blog-backend/wix-blog-backend/guides/blog-intro.md

```diff
@@ -4,6 +4,19 @@ title: Introduction
 
 # Introduction
 
+Wix Blog allows site owners to create and manage blogs. They can create and manage posts, categories and tags on their blog page. Site contributers can also create and manage blogs given the appropriate roles and permissions. 
+
+There are 3 types of blog roles: 
+> - **Blog Editors** can fully manage the blog, but not other areas of a site.
+> - **Blog Writers** can create and publish their own posts on a blog.
+> - **Guest writer** can only create posts. Their posts must be reviewed and published by a site owner or blog editor. 
```

Comment:
Their posts must be reviewed ---> Posts written by guest writers must be reviewed